server/index.js

```diff
@@ -20,18 +27,26 @@ initRoutes(app, db, config);
 if (process.env.NODE_ENV === 'development') {
   seed(app, db);
 }
+// Instance webpack and give the compilers (Configs)
+const compiler = webpack([clientConfig, serverConfig]);
 
-boot(app, db);
+compiler.apply(new FriendlyErrorsWebpackPlugin()); // Just to show better messages and erros
+
+app.use(webpackDevMiddleware(compiler, {
+  serverSideRender: true,
+}));
 
+app.use(
+  webpackHotMiddleware(
+    compiler.compilers.find(c => c.name === 'client'),
+    {
+      log: () => {},
+    },
+  ),
+);
 
-// consign({ verbose: true, cwd: 'server' })
-//   .include('libs/config.js')
-//   .then('db.js')
-//   .then('auth.js')
-//   .then('libs/middlewares.js')
-//   .then('routes')
-//   .then('seed.js')
-//   .then('libs/boot.js')
-//   .into(app);
+app.use(webpackHotServerMiddleware(compiler));
```

Comment:
This one does the magic to handle updates in the server side bundle that generates the code on requests